cmd/thanos/store.go

```diff
@@ -495,6 +495,25 @@ func runStore(
 			s.Shutdown(err)
 		})
 	}
+
+	// Start bloom name filter updater.
+	{
+		ctx, cancel := context.WithCancel(context.Background())
+		level.Debug(logger).Log("msg", "setting up periodic update for label names")
```

Comment:
```suggestion
		level.Info(logger).Log("msg", "setting up periodic update for label names")
```
 I guess info would make sense here